uhabits-android/src/main/res/values/strings.xml

```diff
@@ -228,4 +228,5 @@
     <string name="activity_not_found">No app was found to support this action</string>
     <string name="pref_midnight_delay_title">Extend day a few hours past midnight</string>
     <string name="pref_midnight_delay_description">Wait until 3:00 AM to show a new day. Useful if you typically go to sleep after midnight. Requires app restart.</string>
+    <string name="decimal_separator">.</string>
```

Comment:
I think we can use [DecimalFormatSymbols.getDecimalSeparator](https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormatSymbols.html#getDecimalSeparator()) for this.